stravalib/tests/integration/strava_api_stub.py

```diff
@@ -118,6 +118,10 @@ def method_wrapper(
                 if isinstance(response, list) and n_results is not None:
                     # make sure response has n_results items
                     response = (response * (n_results // len(response) + 1))[:n_results]
+                elif n_results is not None:
+                    # force single response in example into result list (not all examples provide lists)
```

Comment:
This is super nitpicky @jsamoocha BUT [PEP 8](https://peps.python.org/pep-0008/#comments) specifies that comments should start with a capital letter.  i'd love us to follow pep 8 for consistency!

Reply:
Sure, I applied your suggestions; thanks!